spec/spec_helper.rb

```diff
@@ -121,6 +121,12 @@
   Kernel.srand config.seed
 
   config.expect_with(:rspec) { |c| c.syntax = :expect }
+
+  # See https://docs.travis-ci.com/user/common-build-problems/#Ruby%3A-tests-frozen-and-cancelled-after-10-minute-log-silence
```

Comment:
This info shouldn't be in the code as a comment, but on the commit message. Please take a second look at https://brainlessdeveloper.com/2018/02/19/git-best-practices-atomic-commits/
to learn why git history is the reason we all use git instead of svn or cvs